tutorials/inputs/inputevent.rst

```diff
@@ -14,13 +14,23 @@ multiple locations, depending on the purpose.
 
 Here is a quick example, closing your game if the escape key is hit:
 
-::
+.. tabs::
+ .. code-tab:: gdscript GDScript
 
     func _unhandled_input(event):
         if event is InputEventKey:
             if event.pressed and event.scancode == KEY_ESCAPE:
                 get_tree().quit()
 
+ .. code-tab:: csharp
+
+    public override void _UnhandledInput(InputEvent @event)
+    {
+        if (@event is InputEventKey eventKey)
+            if (eventKey.Pressed && eventKey.Scancode == (int)KeyList.Escape)
```

Comment:
It's more like a nitpicking, but probably we could insert a space between `(int)` and `KeyList` for consistency.

Reply:
I checked how casts were written in the [C# section](http://localhost:8000/getting_started/scripting/c_sharp/c_sharp_features.html#type-conversion-and-casting) and there's no space here, so idk.

Reply:
Yeah, I don't have a strong opinion on this one (though I wish we shared the same IP address, so I can see what's on the linked page :stuck_out_tongue: ).

I thought it was the same with Java/Scala convention, since Rider's default formatting configuration inserts a space between a cast and its value. But I just found that VS has a different default value, so I agree that we better leave it as it is.

Reply:
Oh, that link was really wrong. It was obviously this page : https://godot.readthedocs.io/en/3.0/getting_started/scripting/c_sharp/c_sharp_features.html#type-conversion-and-casting